client/src/components/ProfileDetailsCard/ProfileDetailsCard.tsx

```diff
@@ -7,33 +7,26 @@ interface PropTypes {
   profile: Profile;
 }
 
-const ProfileDetailsCard = ({ profile }: PropTypes) => {
-  return (
-    <Card sx={{ width: '50%' }}>
-      <CardMedia image={profile.coverPhoto} title="cover photo" sx={{ minHeight: '200px' }} />
-      <CardContent sx={{ position: 'relative', bottom: '135px', marginBottom: '-135px', padding: '30px' }}>
-        <Grid
-          container
-          direction="column"
-          alignItems="center"
-          justifyContent="space-evenly"
-          sx={{ minHeight: '600px' }}
-        >
-          <Avatar sx={{ border: '4px solid white', width: '170px', height: '170px' }} src={profile.photo} />
-          <Typography variant="h5">{profile.name}</Typography>
-          <Typography>
-            <Room sx={{ margin: '0rem 0.3rem', color: 'primary.main' }} />
-            {profile.city}
-          </Typography>
-          <Typography variant="h6" sx={{ width: '100%' }}>
-            About me
-          </Typography>
-          <Typography>{profile.description}</Typography>
-          <Gallery images={profile.uploadedImages} />
-        </Grid>
-      </CardContent>
-    </Card>
-  );
-};
+const ProfileDetailsCard = ({ profile }: PropTypes) => (
+  <Card sx={{ width: '50%' }}>
+    <CardMedia image={profile.coverPhoto} title="cover photo" sx={{ minHeight: '200px' }} />
+    <CardContent sx={{ position: 'relative', bottom: '135px', marginBottom: '-135px', padding: '30px' }}>
+      <Grid container direction="column" alignItems="center" justifyContent="space-evenly" sx={{ minHeight: '600px' }}>
```

Comment:
Since our `Grid` container does not have any items, might as well consider using a `Box` component here

Reply:
When I tried switching to box, it messed up the layout because box can't do:
`direction="column" alignItems="center" justifyContent="space-evenly"`
so I left it as grid

Reply:
https://mui.com/system/flexbox/#flex-direction

Just for future reference, it needs to be applied slightly different on a Box component 👍🏻